tutorials/parquet-catalog-demos/euclid-hats-parquet.md

```diff
@@ -0,0 +1,235 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Euclid Quick Release 1: MER Catalogs in HATS Parquet
+
++++
+
+## Learning Goals
+By the end of this tutorial, you will:
+
+- # [TODO]
+
++++
+
+## Introduction
+
++++
+
+This notebook explores the HATS version of the [Euclid Q1](https://irsa.ipac.caltech.edu/data/Euclid/docs/overview_q1.html) MER Catalogs that has been created by IRSA.
+In this version, the three MER Catalogs (MER, MER Morphology, and MER Cutouts) have been joined by Object ID into a single Parquet dataset.
+It contains 29,953,430 rows and 601 columns. The total size is 32 GB.
+The dataset is partitioned following [HATS](https://hats.readthedocs.io/) (Hierarchical Adaptive Tiling Scheme).
+
++++
+
+## Installs and imports
+
+```{code-cell}
+# !pip install 'hats>=0.5' 'lsdb>=0.5' matplotlib numpy s3fs
+```
+
+```{code-cell}
+import os
+
+import dask.distributed
+import hats
+import lsdb
+import matplotlib.colors
+import matplotlib.pyplot as plt
+import numpy as np
+```
+
+## 1. Setup
+
+```{code-cell}
+# Need UPath for the testing bucket. Otherwise hats will ignore the credentials that Fornax
+# provides under the hood. Will be unnecessary after the dataset is released in a public bucket.
+from upath import UPath
+
+# AWS S3 path where this dataset is stored.
+s3_bucket = "irsa-fornax-testdata"
+s3_key = "EUCLID/q1/mer_catalogue/hats"
+euclid_s3_path = UPath(f"s3://{s3_bucket}/{s3_key}")
+```
+
+We will use the [`hats`](https://hats.readthedocs.io/) library to visualize the catalog and access the schema.
+
+```{code-cell}
+# Load the parquet dataset using hats.
+euclid_hats = hats.read_hats(euclid_s3_path)
+```
+
+## 2. Visualize the on-sky density of Q1 Objects and HATS partitions
+
++++
+
+Euclid Q1 covers four non-contiguous fields: Euclid Deep Field North (22.9 sq deg), Euclid Deep Field Fornax (12.1 sq deg), Euclid Deep Field South (28.1 sq deg), and LDN1641.
+We can visualize the Object density in the four fields using `hats`.
+
+```{code-cell}
+# Visualize the on-sky distribution of objects in the Q1 MER Catalog.
+hats.inspection.plot_density(euclid_hats)
+```
+
+HATS (Hierarchical Adaptive Tiling Scheme) is a spatial partitioning based on HEALPix that aims to
```

Comment:
I would move this narrative much higher, otherwise it reads a bit more in medias res with the setup and imports, etc.
Or at least consider moving the spelling out of HATS to the previous section where you mention that we will use the `hats` library

Reply:
I moved some of this higher. All of the HATS, `hats`, and `lsdb` parts may still need some rearranging.

Reply:
Did a little more rearranging here so all the descriptions and links are in one paragraph in the intro.